lib/filters/organization_filter.rb

```diff
@@ -0,0 +1,19 @@
+module ActiveInteraction
+  class OrganizationFilter < Filter
+    register :organization
+
+    def cast(value, context)
+      schema = OrganizationUpdateSchema.schema[:properties]
+      included_keys = value.keys.map(&:to_sym)
+      if included_keys.all? {|s| schema.key? s } && valid?(value)
```

Comment:
Yeah oops, it won't work with anything nested. Keep forgetting that's gotta be supported.

`#valid?`is just calling `JSON::Validator.validate(OrganizationUpdateSchema.schema, value)` (down on line 16), which is I believe what you're suggesting? (`OrganizationUpdateSchema` doesn't have a #validate method) 

The other bit is just to ensure that no params that aren't in the schema are included. I didn't actually change how create works, since all params are required. This would for sure be a next step, I just wanted to get this out so updates worked as expected.